### DISCUSSION TITLE
My profile page return 404

### DISCUSSION THREAD

**Discussion by IzetDelibasic** [community/community#112142] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

It seems I'm encountering an issue with my GitHub profile. When I try to access it, I'm getting a 404 "not found" error, and I'm unable to commit. Even when I do commit, nothing shows up.

**Comment by IzetDelibasic** [comment, marked as answer]
Discussion is closed, the profile has been unflagged.

**Reply by AnishThakar** [reply]
im facing the same issue

**Reply by silindokuhleL** [reply]
unflagged? help us fix the issue

**Comment by AnishThakar** [comment]
im facing the same issue

**Comment by silindokuhleL** [comment]
I'm facing the same issue? Can you guys assist 

I can create a PR, but  my teammates can review it, it seems as if my profile is flagged

**Reply by ATOMworkplace** [reply]
hello @silindokuhleL  I can see your profile i think it is unflagged now, can you share your experience how you got your id unflagged I am facing same issue

**Comment by PaleHazeGuy** [comment]
I'm facing the same issue, I just created an organization and then my profile got 404

**Reply by ATOMworkplace** [reply]
Hi @PaleHazeGuy  It seems your issue is fixed i can see your profile, I raised a ticket its been a day now, can you tell how you got unflagged?

**Comment by vivekvardhan808** [comment]
I'm facing the same issue, I just created an organization and then my profile got 404

**Comment by jukent** [comment]
I am facing the same issue. I see lots of posts about this, but no instructions on how to un-flag it. Do I just have to wait?

**Comment by asamimi-chan** [comment]
Same here. My profile returns a 404 error page and there does not seem to be a documented method to fix this issue.
